packages/test/src/MockResolver.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  DispatchContext,
+  ReceiveAction,
+  actionTypes,
+  ActionTypes,
+} from '@rest-hooks/core';
+import { useCallback, useContext, useMemo } from 'react';
+import React from 'react';
+
+import { Fixture, actionFromFixture } from './mockState';
+
+type Props = {
+  children: React.ReactNode;
+  fixtures: Fixture[];
+};
+
+/** Can be used to mock responses based on fixtures provided.
+ *
+ * <MockResolver fixtures={postFixtures[state]}><MyComponent /></MockResolver>
+ *
+ * Place below <CacheProvider /> and above any components you want to mock.
+ */
+export default function MockResolver({ children, fixtures }: Props) {
+  const fetchToReceiveAction = useMemo(() => {
+    const actionMap: Record<string, ReceiveAction> = {};
+    for (const fixture of fixtures) {
+      const { key, action } = actionFromFixture(fixture);
+      actionMap[key] = action;
+    }
+    return actionMap;
```

Comment:
```suggestion
    const actionMap: Record<string, ReceiveAction> = fromEntries(fixtures.map((fixture) => {
      const { key, action } = actionFromFixture(fixture);
      return [key, action];
    }));
    return actionMap;
```
with `import fromEntries from 'object.fromentries';`, but i understand if you prefer this approach to adding the dep, despite being very useful and clean :-)

Reply:
ya keeping the library small is a primary objective.